R/rolling_cfr.R

```diff
@@ -0,0 +1,124 @@
+#' Rolling case fatality rate
```

Comment:
'Rolling CFR' is a bit ambigious (as mentioned here: https://github.com/epiverse-trace/datadelay/issues/4) so have edited README and need to be precise in documentation that this is a static CFR calculated up to different points in time, rather than a time-varying CFR.